docs/README.md

```diff
@@ -0,0 +1,64 @@
+# HyperFormula documentation
+
+HyperFormula comes with a dedicated, regularly-updated documentation portal.
+
+View the documentation's latest production version at https://handsontable.github.io/hyperformula/.
+
+## About HyperFormula documentation
+
+The HyperFormula documentation is built with [VuePress](https://vuepress.vuejs.org/), a Vue-powered Static Site Generator.
+
+Our VuePress setup:
+
+* Publishes the docs as a sleek, responsive single-page application (SPA).
+* Boasts a great search engine.
+* Lets us write the docs in Markdown (and use Vue inside Markdown!).
+* Lets us quickly build the docs output onto a local server.
+* [COMING SOON] Versions and deploys the docs together with the software.
```

Comment:
IMO we should not describe VuePress, for me it's an internal implementation detail, and we may change the engine behind the docs in the future. But, we can have a short mention because some editing features are based on VuePress capabilities so the editor may use features described here https://vuepress.vuejs.org/guide/markdown.html